cmd/describe/cluster/cmd.go

```diff
@@ -179,6 +179,15 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
+	workloadMonitoringDisabled, ok := cluster.GetDisableUserWorkloadMonitoring()
+	isWorkloadMonitoringEnabled := "Enabled"
+	if !ok {
+		isWorkloadMonitoringEnabled = "Unknown"
```

Comment:
is "Unkonwn" a valid value? 
Another option is to return the default value("Enabled") if cannot get the value.

Reply:
I agree with @marcolan018 `unknown` is not a valid value and I would rather not show anything than to show an invalid value. 
Same as we can not just go with Enabled if the value is not found. 

Lets just prefix it to `str` if we have the value, something like 
```
	workloadMonitoringDisabled, ok := cluster.GetDisableUserWorkloadMonitoring()
	if ok {
		isWorkloadMonitoringEnabled := "Enabled"
		if workloadMonitoringDisabled {
			isWorkloadMonitoringEnabled = "Disabled"
		}
		str = fmt.Sprintf("Workload Monitoring:        %s\n", isWorkloadMonitoringEnabled)	
	}
```

Reply:
changed

Reply:
Can you update this again, so we are not passing in an empty string to `str` similar to the example in the previous comment. 

We can prefix the workload monitoring to `str`

Reply:
updated